conjureup/controllers/newcloud/gui.py

```diff
@@ -89,21 +86,21 @@ def __post_bootstrap_exec(self):
                                               _post_bootstrap_sh,
                                               shell=True,
                                               env=app.env),
-                                      self.__handle_exception)
+                                      handle_exception)
                 future.add_done_callback(self.__post_bootstrap_done)
             except Exception as e:
-                return self.__handle_exception(e)
+                return handle_exception(e)
 
     def __post_bootstrap_done(self, future):
         try:
             result = json.loads(future.result().decode('utf8'))
         except Exception as e:
-            return self.__handle_exception(e)
+            return handle_exception(e)
```

Comment:
While we're changing this, we should really add more detail to this error specifically.
JSON parsing errors are terrible, they say shit like "expected : in line 1" when what we really want to see is the text it tried to parse, because it's usually a real error message that would've been helpful. Maybe instead of just passing `e` through, we could build a new exception that included the actual decoded result?